launch_frontend/launch_frontend/parser.py

```diff
@@ -0,0 +1,213 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Module for Parser methods."""
+
+import launch
+
+import launch_ros
+
+from .entity import Entity
+
+
+def get_attr_or_default(entity, attr, default_value):
```

Comment:
@ivanpauno plain ol' [`getattr()`](https://docs.python.org/3/library/functions.html#getattr) has this exact same behavior.